docs/configuration.md

```diff
@@ -345,7 +345,13 @@ _Default: Infinity_
 
 This parameter configures the maximum zoom level available. Set it to a lower value to prevent your users from zooming in too deeply.
 
-### numberDisplayedTicks
+### restrictpan
```

Comment:
why not use camelCase like for the other parameters?

Reply:
Ah, I had a bit of difficulty getting the GitHub anchors to work. Just tested this in another repository after looking at how GitHub does the conversion of headers to HTML and will fix later today.
It is referenced in README.md, which is why I was trying to make the anchor work (open file at the correct place) and not just open the file.